### ISSUE TITLE
Optimized sequence of commands for BIOMED-2 primer set (docs)

### ISSUE THREAD

**Issue by dbolotin** [milaboratory/mixcr#227]

**Comment by suntaosimon** [comment]
excuse me. I see the (doc) label on this topic but where is the actual doc file?

**Comment by dbolotin** [comment]
Yes, default settings are already optimal for CDR3 extraction from any type of BIOMED-2 data. This issue devoted for optimization of analysis parameters for long IGH set of BIOMED-2 primers, to simplify exclusion of primer sequences from consideration, and pre optimize maximal possible clonalSequence.
